clang/lib/DPCT/SaveNewFiles.h

```diff
@@ -71,8 +69,11 @@ void rewriteFileName(clang::tooling::UnifiedPath &FileName);
 // extension needs to wait until all replacements are generated to get the
 // correct result.
 void rewriteFileName(clang::tooling::UnifiedPath &FileName,
-                     const clang::tooling::UnifiedPath& FullPathName);
+                     const clang::tooling::UnifiedPath &FullPathName);
```

Comment:
Normally, it's better to keep the PR as small as possible on footprint by not including format code.  (this will make git blame works well).

Reply:
I agree! 

At the same time, it will be helpful if we format the project with clang-format in a new PR. This will ease the formatting the code before pushing the commit. Your thoughts?